core/src/main/resources/hudson/model/View/builds.jelly

```diff
@@ -27,6 +27,7 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
   <l:layout title="${it.displayName}">
     <st:include page="sidepanel.jelly" />
+    <l:breadcrumb title="${%buildHistory(it.class.name=='hudson.model.AllView' ? app.displayName : it.displayName)}" />
```

Comment:
I don't think the page title here is appropriate for the breadcrumb, "Build History of Jenkins" (primary view at least).
really the title should probably be this as well:
```suggestion
    <l:breadcrumb title="${%Build History}" />
```